src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperService.cs

```diff
@@ -16,13 +18,17 @@ namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
     /// <summary>
     /// Implements <see cref="IDistributedCacheTagHelperService"/> and ensure
     /// multiple concurrent requests are gated.
+    /// The entries are stored like this:
```

Comment:
This probably belongs on a <remarks></remarks> section and I'm not sure about using - to define "buillet points", I don't think we do it anywhere and it will most likely not be picked up by the documentation